src/main/java/thaumicboots/item/boots/voidwalker/ItemElectricVoidwalkerBoots.java

```diff
@@ -21,91 +11,63 @@
 import cpw.mods.fml.relauncher.Side;
 import cpw.mods.fml.relauncher.SideOnly;
 import ic2.api.item.ElectricItem;
-import ic2.api.item.IElectricItem;
 import taintedmagic.common.registry.ItemRegistry;
-import thaumcraft.api.IRepairable;
-import thaumcraft.api.IRunicArmor;
-import thaumcraft.api.IVisDiscountGear;
 import thaumcraft.api.IWarpingGear;
-import thaumcraft.api.aspects.Aspect;
 import thaumcraft.client.fx.ParticleEngine;
 import thaumcraft.client.fx.particles.FXWispEG;
 import thaumcraft.common.Thaumcraft;
 import thaumcraft.common.items.armor.Hover;
+import thaumicboots.api.ItemElectricBoots;
 import thaumicboots.main.utils.TabThaumicBoots;
 
-public class ItemElectricVoidwalkerBoots extends ItemArmor
-        implements IVisDiscountGear, IWarpingGear, IRunicArmor, IRepairable, ISpecialArmor, IElectricItem {
-
-    public int maxCharge = 1000000;
-    public int energyPerDamage = 500;
-    public int visDiscount = 5 + 2;
-
-    public double transferLimit = 400;
+public class ItemElectricVoidwalkerBoots extends ItemElectricBoots implements IWarpingGear, ISpecialArmor {
 
     public ItemElectricVoidwalkerBoots(final ArmorMaterial material, final int j, final int k) {
         super(material, j, k);
         setCreativeTab(TabThaumicBoots.tabThaumicBoots);
-        setUnlocalizedName("ItemElectricVoid");
-        setTextureName("thaumicboots:electricVoid_16x");
-
+        setUnlocalizedName(unlocalisedName);
```

Comment:
dead code

Reply:
That's not surprisingly, if that's not that it doesn't have a name and causes an error

Reply:
You're setting `unlocalizedName` in `setBootsData()`, which is what `setUnlocalizedName()` does too, thereby making this redundant.